torch/jit/annotations.py

```diff
@@ -305,6 +305,8 @@ def try_ann_to_type(ann, loc):
         return FutureType(try_ann_to_type(ann.__args__[0], loc))
     if ann is float:
         return FloatType.get()
+    if inspect.isclass(ann) and issubclass(ann, builtins.complex):
+        return ComplexDoubleType.get()
```

Comment:
Another dumb question, why is this not `ann is complex`?

Reply:
Also, is there a usecase for creating subclasses of `complex`?

Reply:
complex is a python class hence the `isclass` call. umm no there's no use specific case of creating subclasses of complex

Reply:
But aren't `int` and `float` classes too?

Reply:
ok wait you are right `ann is complex` works!